google/cloud/apigeeregistry/v1/check/response.proto

```diff
@@ -0,0 +1,85 @@
+// Copyright 2023 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+// (-- api-linter: core::0215::versioned-packages=disabled
+//     aip.dev/not-precedent: Support protos for the apigeeregistry.v1 API. --)
+package google.cloud.apigeeregistry.v1.style;
+
+import "google/api/field_behavior.proto";
+import "google/protobuf/timestamp.proto";
+
+option java_package = "com.google.cloud.apigeeregistry.v1.style";
+option java_multiple_files = true;
+option java_outer_classname = "CheckResponseProto";
+option go_package = "github.com/apigee/registry/rpc;rpc";
+
+// Response is the results of running the check command.
+message Response {
```

Comment:
This is great - can we align this name with [ConformanceReport](https://github.com/apigee/registry/blob/0c23b1bced76ebfdd7d8a4e420659dc54da5ebab/google/cloud/apigeeregistry/v1/style/conformance_report.proto#L33)? (is it similar?)

Reply:
done